### DISCUSSION TITLE
Connection Tile View, Single Panel

### DISCUSSION THREAD

**Discussion by TheBrambleShark** [mRemoteNG/mRemoteNG#2560] [Q&A]
Hi all, coming over from RDCM. One of the features I used very commonly there was the folder tile view. That is, in the connections menu, if you select a folder it shows a preview of all of the server connections in that folder. This allowed me to effectively monitor multiple machines at a time, albeit through a thumbnail-sized screen.

I'm looking to replicate this with mRemoteNG. The panels feature seems to be intended for this (production panel, development panel, etc.). Is there a way to view multiple connections in a panel at the same time, preferably tiled, or do I need to create multiple panels for this? If it's the latter, how can I "hide" connection views? I don't want to close the connection, I just don't want to see it. This is akin to switching tabs in a panel - doesn't disconnect when you select another tab, you just don't see the contents of that tab.

**Comment by laurNtiu** [comment]
"Easy" workaround:
1) Open left connection
2) [Optional] Rename panel (if left and right are in the same panel)
3) Open right connection
4) Drag panel of right connection to create a side-by-side view
